libkbfs/folder_block_manager.go

```diff
@@ -1306,25 +1306,36 @@ func (fbm *folderBlockManager) clearLastQRData() {
 	fbm.lastReclamationTime = time.Time{}
 }
 
-func (fbm *folderBlockManager) doCleanSyncCache() (err error) {
-	defer fbm.cleanSyncCacheGroup.Done()
+func (fbm *folderBlockManager) doCleanDiskCaches() (err error) {
+	defer fbm.cleanDiskCachesGroup.Done()
+	// TODO: determine if these can be parallelized
```

Comment:
From a correctness standpoint, parallelizing them should be fine.  Not sure if it will make the disk performance extra sad though.  I'd err on the side of keeping it serialized for now, unless we have a need to speed it up.